.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
This might be placed better in `.git/info/exclude` if it's done for personal stuff

Reply:
See https://help.github.com/articles/ignoring-files/

Reply:
OK